Makefile

```diff
@@ -62,7 +64,8 @@ endif
 LDFLAGS += -X ${BASE_PKG}/internal/version.commit=${GIT_COMMIT}
 LDFLAGS += -X ${BASE_PKG}/internal/version.treestate=${GIT_DIRTY}
 
-IMAGE_TAG := ${GIT_BRANCH}
+# Replaces all "/" in git branch name with "-": docker tags can not contain a "/".
+IMAGE_TAG := $(shell echo $(GIT_BRANCH) | sed 's/\//-/g')
```

Comment:
Just as a curiosity, you can use a different separator as well: "sed 's#/#-#g'". Just for info :)

Reply:
Interesting I had some awkward issue with make, whereby this doesn't seem to work.

If I use `IMAGE_TAG := $(shell echo $(GIT_BRANCH) | sed 's#/#-#g')`, I get the following error `Makefile:68: *** unterminated call to function `shell': missing `)'.  Stop`.

Rather than spending lots of time debugging I opted for the more confusing separator and adding a comment to explain.